truthbeam/config_test.go

```diff
@@ -0,0 +1,92 @@
+package truthbeam
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/config/confighttp"
+)
+
+// TestConfigValidate tests the Validate method of the Config struct
+func TestConfigValidate(t *testing.T) {
+	tests := []struct {
+		name        string
+		config      *Config
+		expectError bool
+		errorMsg    string
+	}{
+		{
+			name:        "empty endpoint should fail",
+			config:      &Config{},
+			expectError: true,
+			errorMsg:    "endpoint must be specified",
+		},
+		{
+			name: "valid endpoint should pass",
+			config: &Config{
+				ClientConfig: confighttp.ClientConfig{
+					Endpoint: "http://example.com",
```

Comment:
What about other possible values such as:
- lunatic://absent.example.com
- file:///tmp/
- http://example.com:70000

Worth to test? I am not sure what is already validated on OpenTelemetry side.